src/styles/helpers/color/validateColor.js

```diff
@@ -0,0 +1,5 @@
+export default function validateColor(color) {
```

Comment:
it looks good to me. I would just name the function `isValidColor` then in styled component I would do 
```
const brandMainColor = isValidColor(props.color) ? props.color : brand.main;
```

what do you think? :)

Reply:
done!